DataFormats/Common/test/testMultiAssociation.cc

```diff
@@ -640,11 +640,11 @@ bool testMultiAssociation::tryBadFill(int i) {
     } break;
     case 11: {  // Can copy a LazyFiller, but crash if I fill twice
       MultiRef::LazyFiller filler = m.lazyFiller(handleK1, true);
-      MultiRef::LazyFiller filler2 = filler;
+      const MultiRef::LazyFiller& filler2 = filler;
```

Comment:
Apparently we need to add 
```suggestion
      const MultiRef::LazyFiller& filler2 = filler; //NOLINT
```
here.

@smuzaffar @mrodozov Should that be done in a separate PR and then we refresh this one?